src/styles/main.scss

```diff
@@ -1,3 +1,33 @@
+@import "./variables";
+@import "./calendar";
+
 body {
+  display: flex;
+  justify-content: center;
   margin: 0;
+  margin-top: 25px;
+}
+
+@for $colum from 1 through 31 {
+  .calendar__day:nth-child(#{$colum})::before {
+    content: "#{$colum}";
+  }
+}
+
+@each $day, $number in $days {
+  .calendar--start-day-#{$day} {
+    &__day:first-child {
+      margin-left: ($day-size + $gap);
+    }
+  }
+}
+
+@for $colum from 28 through 31 {
```

Comment:
https://github.com/mate-academy/layout_calendar/assets/53228259/92125b07-1854-4113-a9af-b2db29f16427

you circles aren't working